cw/util/trace_library.py

```diff
@@ -0,0 +1,194 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+import datetime
+from dataclasses import asdict, dataclass
+from typing import Optional
+
+import numpy as np
+import sqlalchemy as db
+import sqlalchemy_utils as db_utils
+from sqlalchemy.orm import sessionmaker
+
+
+@dataclass
+class Metadata:
+    """ Metadata for capture.
+
+    Stores information about the capture of the traces.
+
+    """
+    config: str
+    datetime: datetime.datetime
+    bitstream: str
+    binary: str
```

Comment:
Hm usually it is not a good practice to store large binary blobs (such as the actual firmware binary) into a database. However, as we do not really care about the database size, it might be an option. Do we want to discuss this in the next SCA/FI meeting?
Commit of the ot-sca repo is a good idea, I'll add it.

Reply:
Good idea to discuss this today in the SCA meeting.

Reply:
Based on our discussion, I added fields for storing the binary and the bitstream.